ETC.tsv

```diff
@@ -17665,41 +17665,41 @@ ETC_20151224_017666	In substance, this is the kind of Golem and it is a creature
+ETC_20151224_017688	In various regions, they created either the Kupoles or the idols of the spirits. Some of them started to work as evil creatures after Medzio Diena.
 ETC_20151224_017689	Stonacons are originated from the plant seeds which chipmunk type monsters forgot after buring them under the ground.
-ETC_20151224_017690	They are the monsters that were created from the transformations of stone architectures that were engraved with the letters of magic or Runes.
-ETC_20151224_017691	It is a subspecies of Drapeliun.
+ETC_20151224_017690	They are the monsters that were created from the transformations of stone architectures that were engraved with the letters of magic or runes.
+ETC_20151224_017691	A subspecies of Drapeliun.
 ETC_20151224_017692	It is still to be investigated how the monsters that were created from the seeds of the plants have the appearance of snake.
-ETC_20151224_017693	Long time ago, Elet used to be a fairy or a soul. They received the curse of the goddess, Ausrine and they hid into the human world and lived on. Since Medizo Diena, when the demons invaded the world, they are showing their determination to participate.
-ETC_20151224_017694	What's unique about Nuo is that they are subordinates of the demon king, Kartas. Although Kartas possesses the brutal and the unexpected personality, it is so incredible that Nuo who is under the control of Kartas possesses the power and the number for the ones who know well about the circumstances in the demons.
-ETC_20151224_017695	Nuka is under the control of the demon king, Baiga and he is dispatched here to do the task which Gesti received from Giltine. But, Gesti is controlling them unlike what Baiga once thought.
-ETC_20151224_017696	Elma is one of the Gesti's bodyguards and she is dispatched to various duties and she also works as the guard at the devildom.
+ETC_20151224_017693	A long time ago, Elet used to be a fairy or a soul. They received the curse of Goddess Ausrine and they hid into the human world and lived on. Since Medzio Diena, when the demons invaded the world, they are showing their determination to participate with the other demons.
+ETC_20151224_017694	What's unique about Nuo is that they are subordinates of the Demon King Kartas. Although Kartas possesses a brutal and unexpected personality, it is so incredible that Nuo who is under the control of Kartas possesses the power and the number for the ones who know well about the circumstances in the demons.
+ETC_20151224_017695	Nuka is under the control of the Demon King Vaiga and was dispatched here to fulfill the tasks that Gesti received from Giltine. But, Gesti is controlling them unlike what Vaiga once thought.
+ETC_20151224_017696	Elma is one of the Gesti's bodyguards and she is dispatched to various duties and she also works as the guard at the devil realm.
```

Comment:
Elma is one of the Gesti's bodyguards and she is dispatched to various duties and she also works as the guard at the devil realm.

Elma is one **of Gesti's bodyguards, she** is dispatched to various duties **and also works** as **a guard at** the devil realm.